client/js/commands/search.js

```diff
@@ -4,10 +4,15 @@ import store from "../store";
 import {router} from "../router";
 
 function input(args) {
+	if (!store.state.settings.searchEnabled) {
+		return true;
```

Comment:
I wonder if just returning false here would be reasonable. We'd end up behaving as if the command is completely unknown, which seems reasonable given that we're stripping its existence from the UI while its disabled.